tests/gnmi/helper.py

```diff
@@ -177,3 +177,17 @@ def gnoi_reboot(duthost, localhost, method, delay, message):
         return -1, output['stderr']
     else:
         return 0, output['stdout']
+
+
+def gnoi_killprocess(duthost, localhost, request_json_data):
+    env = GNMIEnvironment(duthost, GNMIEnvironment.GNMI_MODE)
+    ip = duthost.mgmt_ip
+    port = env.gnmi_port
+    cmd = "gnmi/gnoi_client -target %s:%s " % (ip, port)
```

Comment:
Would it make sense to create a helper function that will generate a gnoi client command? Seems like the binary and the flags like certs and logtostderr would stay the same for the most part for all current and future rpc commands.

Reply:
Thanks, I've changed to move gnoi_client in a helper function